doc/reviewing_PR_guide.md

```diff
@@ -0,0 +1,91 @@
+
+## 2. Length restrictions
+
+The length restrictions are just a suggestion, but an important one.  If a PR goes longer than this, there's a high chance it violates rule 1.  If it does not violate rule 1 and has good reason for being very long (eg. it's a simple automated change to a ton of documents), then this one might be skipped.  If the changes are very tidily broken into organized commits, and those commits are not squashed during merge, then it might also be safe to ignore this one.  Mapgen changes are also pretty much always exempt because maps are huge... but if you have a lot of other changes, it might be a good idea to do the maps in a separate PR first.
+
+The reason for this rule is similar to #1.  Very long PRs have a higher chance of introducing errors, and then those errors are hard to find.  Reviewers and bugfixers are human, and the longer the single block of changes, the harder it is to find the one that introduced the problem.
+
+Once again though, because it's caused problems before, some changes *can't* be broken into more than one PR, and that's okay if it's sufficiently justified.  In particular, we don't like dead code, so if a PR is long because it implements a new system and also adds a map or a monster that uses the new system, that's usually preferable to adding the map/monster in a follow-up.
+
+PRs that break this rule are likely to take longer to merge because they're going to be very hard to review, but we will eventually get to them.
+
+## 3. Outside sourced material
+
+CDDA is licensed under CC-BY-SA 4.0.  Unless you're really, really certain you understand *exactly* how this works, then you should not approve any PR using **any** material from outside the project: not even things like names, obvious references, et cetera.  Don't accept someone else's explanation for how it works, don't assume something is too trivial to count, and don't assume the material is fair use.  Just flag it as not meeting this criterion.
+
+Failing this guideline doesn't mean your PR is rejected, it just means we need to take a closer look.  The lead devs have been forced to become pretty expert at this and we'd like to ensure our rules are followed consistently, so if this is flagged, we'll come in and assess if it's okay.
+
+
+## 4. Changes consistent with the description
+
+While the PR description doesn't necessarily need to exhaustively outline every single line of code change, reading the PR description should clearly tell the reviewer what they're going to see in the changes.  There should not be any surprises in there.  If something was changed after the description was written, it needs to be updated.
+
+I find it helpful to think: if I was coming back to this PR in a year trying to remember what was in it to see if it might be responsible for a bug, could I use the PR description to figure that out?  If I *have* to read the code changes to even know if I'm in the right spot, there's a chance the description needs work.
```

Comment:
First person inconsistency again.

Reply:
```suggestion
I find it helpful to think: if I was coming back to this PR in a year trying to remember what was in it to see if it might be responsible for a bug, could I use the PR description to figure that out?  If a person coming later *has* to read the code changes to even know if they're in the right spot, there's a chance the description needs work.
```